sdk/python/kfp/cli/recurring_run.py

```diff
@@ -89,7 +89,6 @@ def create(ctx: click.Context,
            end_time: Optional[str] = None,
            interval_second: Optional[int] = None,
            max_concurrency: Optional[int] = None,
-           params: Optional[dict] = None,
```

Comment:
Looks like this was never used, but `args` is? I wonder if we're consistent on using `params` vs. `args`.

Reply:
I think that's right.

Are you referring to consistency between the CLI argument names and the Client parameter names? If so, we definitely aren't consistent everywhere. As a final step in the V2 CLI migration, I'm going to see what can be done to remedy this in a backward compatible way.